WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -147,6 +147,12 @@ private extension CartView {
             orderState: posModel.orderState,
             paymentState: posModel.paymentState)
     }
+
+    var shouldShowClearCartButton: Bool {
```

Comment:
When using `ObservableObject` before I used to worry about derived variables not triggering redraws when the source `@Published var`'s change, looks like this isn't a concern in the aggregated model architecture anymore.

Reply:
We'll have to see how that goes when we shift to `Observation`, which is more granular. It should still track access via computed vars, though the sessions talk more about computeds on the `@Observable`, not on the view.

For `ObservableObject`, you don't really have to worry. Any view which has a dependency (marked with `@ObservedObject`, `@StateObject`, or `@EnvironmentObject`) will be redrawn whenever any published property on that object changes.